dbt_airflow_factory/k8s/k8s_parameters.py

```diff
@@ -39,6 +39,9 @@ class KubernetesExecutionParameters:
     :param is_delete_operator_pod: What to do when the pod reaches its final
         state, or the execution is interrupted. If True: delete the pod.
     :type is_delete_operator_pod: bool
+    :param config_file: The path to the Kubernetes config file. (templated)
+        If not specified, default value is ``~/.kube/config``
+    :type config_file: str
```

Comment:
maybe the `Optional` type should be reflected here as well?

Reply:
good catch I will push update